File: PHP/fizzbuzz.php
<?php

function fizzbuzz($from, $to) {
    for ($i = $form; $i <= $to; $i += 1) {
        if ($i % 15 == 0) {
            echo "FizzBuzz\n";
        } elseif ($i % 5 == 0) {
            echo "Buzz\n";
        } elseif ($i % 3 == 0) {
            echo "Fizz\n";
        } else {
            echo "$i\n";
        }
    }
}

fizzbuzz(1, 100);

File: C++/fizzbuzz.cpp
#include <iostream>

int main()
{
    for (int i = 1; i <= 100; i += 1) {
        if (i % 15 == 0) {
            std::cout << "FizzBuzz";
        } else if (i % 5 == 0) {
            std::cout << "Buzz";
        } else if (i % 3 == 0) {
            std::cout << "Fizz";
        } else {
            std::cout << i;
        }
        std::cout << std::endl;
    }
}
